tutorial/debezium-db2-init/db2server/asncdcaddremove.sql

```diff
@@ -0,0 +1,199 @@
+
```

Comment:
Could you add a comment at the beginning of the file describing its general purpose?